### PR TITLE
Code refactoring for test coverage improvement 

### PR DESCRIPTION
**Pull request by dearrudam**
- I did a little code refactoring in order to be able to improve the test coverage of the `org.eclipse.jnosql.mapping.validation.MappingValidator` class;
- Setting up the Jacoco Maven Plugin to ignore config classes;

### CODE REVIEW

**Comment by dearrudam** [comment]
@otaviojava, could you review this PR?

**Comment by genie-jnosql** [comment]
Can one of the admins verify this patch?

File: jnosql-mapping/jnosql-mapping-validation/src/main/java/org/eclipse/jnosql/mapping/validation/MappingValidator.java [hunk 1]
```diff
@@ -84,16 +84,40 @@ public void validate(ConstructorEvent event) {
...
-            ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
-            return factory.getValidator();
-        }
+    /**
+     * Defines the {@link Validator} that it's going to be used during the validation process
+     *
+     * @throws NullPointerException when cannot to get the validator from CDI
+     */
+    @PostConstruct
+    public void postConstruct() {
```
Comment (otaviojava) [inline jnosql-mapping/jnosql-mapping-validation/src/main/java/org/eclipse/jnosql/mapping/validation/MappingValidator.java:93]:
```suggestion
 void postConstruct() {
```
I guess there is no issue to don't defining it as public.

Comment (otaviojava) [reply jnosql-mapping/jnosql-mapping-validation/src/main/java/org/eclipse/jnosql/mapping/validation/MappingValidator.java:93]: @dearrudam you have the option to commit the suggestion; it only requires an accept on UI.

Comment (dearrudam) [reply jnosql-mapping/jnosql-mapping-validation/src/main/java/org/eclipse/jnosql/mapping/validation/MappingValidator.java]: @otaviojava I learned one more!!! Thanks!!!

**Comment by dearrudam** [comment]
@otaviojava could you go ahead?

**Pull request merged**